inframanager/api_handler/api_handler.go

```diff
@@ -405,8 +458,9 @@ func (s *ApiServer) DeleteNetwork(ctx context.Context, in *proto.DeleteNetworkRe
 		out.Successful = false
 		return out, err
 	}
+	epEntry := entry.(store.EndPoint)
 
-	if err = p4.DeleteCniRules(ctx, server.p4RtC, macAddr, ipAddr); err != nil {
+	if err = p4.DeleteCniRules(ctx, server.p4RtC, epEntry); err != nil {
```

Comment:
Same comment as above ---- merge or handle two different signatures.

Reply:
Leaving this comment for Venkatest to address as separate commit on this branch.

Reply:
here too